src/python-common/ceph/deployment/hostspec.py

```diff
@@ -13,7 +16,8 @@ def assert_valid_host(name: str) -> None:
             assert len(part) <= 63, '.-delimited name component must not be more than 63 chars'
             assert p.match(part), 'name component must include only a-z, 0-9, and -'
     except AssertionError as e:
-        raise SpecValidationError(str(e) + f'. Got "{name}"')
+        logging.error(str(e) + f'. Got "{name}"')
```

Comment:
Can't the caller handle the exception and show a particularly useful message string?

Reply:
Discussed this with @neesingh-rh - caller should handle `SpecValidationError` and return error strings.